examples/resources/credential/cluster_proxy.tf

```diff
@@ -30,3 +31,31 @@ resource "tanzu-mission-control_credential" "proxy_cred" {
     }
   }
 }
+
+# Create transparent cluster proxy credential
+resource "tanzu-mission-control_credential" "transparent_proxy_cred" {
+  name = "transparent_proxy_cred"
+
+  meta {
+    description = "transparent proxy credential"
```

Comment:
nit: Please take care of the indentation throughout tf file.

Reply:
Done